ledger-api/rs-grpc-akka/src/main/scala/com/digitalasset/grpc/adapter/AkkaExecutionSequencerPool.scala

```diff
@@ -15,7 +15,7 @@ import scala.concurrent.{Await, ExecutionContext, Future}
 class AkkaExecutionSequencerPool(
     poolName: String,
     actorCount: Int = AkkaExecutionSequencerPool.defaultActorCount,
-    terminationTimeout: FiniteDuration = 30.seconds,
+    terminationTimeout: FiniteDuration = 180.seconds,
```

Comment:
I think you also need to increase the timeout in [ApiPackageManagementService.scala](https://github.com/digital-asset/daml/blob/d48e9d251d22ba0aea3de2cd51755e3be447e8a1/ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala#L56).

Reply:
Do I actually need it here ?

Reply:
I don't think so, but I might be wrong. I suggest testing it with a very large DAR.

Reply:
reverted.